tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_homescreen_change_wallpaper.py

```diff
@@ -18,7 +20,8 @@ def test_homescreen_change_wallpaper_from_gallery(self):
         https://moztrap.mozilla.org/manage/case/1902/
         reusing /functional/homescreen/test_homescreen_change_wallpaper.py script
         """
-
+        # wait until the homescreen is fully drawn
+        time.sleep(3)
```

Comment:
I guess that's necessary. But I would think there is also some way to detect that the homescreen is fully drawn.